### DISCUSSION TITLE
实时系统FreeRTOS的必要性，以及c++语言的使用

### DISCUSSION THREAD

**Discussion by gengjile** [peng-zhihui/ElectronBot#259] [Q&A]
1. 我看固件STM32代码是使用了FreeRTOS的，但实际上代码里面就只有一个任务。  那这还有使用FreeRTOS的必要吗？
2. 为什么代码里面会夹杂着c++呢？   使用c++的原因是什么？
3. 原理图中好像没有给出摄像头的接口啊？ 难道是飞线连接的？  如果是飞线，为啥不做个接口而要去飞线呢？
![image](https://github.com/peng-zhihui/ElectronBot/assets/150111228/62646253-a3fd-48f9-bed7-f525eb1f0a61)

这是目前我的一些问题。

**Comment by Kitem** [comment]
您好，我是焕挺，您的邮件已经收到，我会尽快回复！

**Comment by VendettaMask** [comment]
来信已收，谢谢！

**Comment by BLDC888** [comment]
您好：你的来信我已收到，我会尽快处理的。
